docs/source/reference/compilation.rst

```diff
@@ -102,6 +102,20 @@ JIT functions
 
       .. seealso:: :ref:`architecture`
 
+   .. method:: inspect_llvm(signature=None)
+
+      Return a dictionary keying compiled function signatures to the human 
+      readable llvm byte code generated for the function.  If the signature 
```

Comment:
I didn't get to this one yet.  I said human-readable to emphasize that it was the human friendly version of the bytecode, as opposed to the usual thing you find in a .bc file.